android/app/src/main/AndroidManifest.xml

```diff
@@ -5,60 +5,63 @@
-        android:windowSoftInputMode="adjustResize">
-        <intent-filter>
-            <action android:name="android.intent.action.MAIN" />
-            <category android:name="android.intent.category.LAUNCHER" />
-        </intent-filter>
-      </activity>
-      <activity android:name="com.facebook.react.devsupport.DevSettingsActivity" />
+        android:icon="@mipmap/ic_launcher"
+        android:theme="@style/AppTheme">
+        <activity
+            android:name=".MainActivity"
+            android:label="@string/app_name"
+            android:configChanges="keyboard|keyboardHidden|orientation|screenSize"
+            android:windowSoftInputMode="adjustResize">
+            <intent-filter>
+                <action android:name="android.intent.action.MAIN" />
+                <category android:name="android.intent.category.LAUNCHER" />
+            </intent-filter>
+        </activity>
+        <activity android:name="com.facebook.react.devsupport.DevSettingsActivity" />
 
 
 
-			<receiver
-		   	android:name="com.google.android.gms.gcm.GcmReceiver"
-		   	android:exported="true"
-		   	android:permission="com.google.android.c2dm.permission.SEND" >
-		   	<intent-filter>
-		       	<action android:name="com.google.android.c2dm.intent.RECEIVE" />
-		       	<category android:name="${applicationId}" />
-		   	</intent-filter>
-			</receiver>
+        <receiver
+            android:name="com.google.android.gms.gcm.GcmReceiver"
+            android:exported="true"
+            android:permission="com.google.android.c2dm.permission.SEND" >
+            <intent-filter>
+                <action android:name="com.google.android.c2dm.intent.RECEIVE" />
+                <category android:name="${applicationId}" />
+            </intent-filter>
+        </receiver>
 
-			<receiver android:name="com.dieam.reactnativepushnotification.modules.RNPushNotificationPublisher" />
-			<service android:name="com.dieam.reactnativepushnotification.modules.RNPushNotificationRegistrationService"/>
-			<service
-		   	android:name="com.dieam.reactnativepushnotification.modules.RNPushNotificationListenerService"
-		   	android:exported="false" >
-		   	<intent-filter>
-		       	<action android:name="com.google.android.c2dm.intent.RECEIVE" />
-		   	</intent-filter>
-			</service>
+        <receiver android:name="com.dieam.reactnativepushnotification.modules.RNPushNotificationPublisher" />
+        <service android:name="com.dieam.reactnativepushnotification.modules.RNPushNotificationRegistrationService"/>
+        <service
+            android:name="com.dieam.reactnativepushnotification.modules.RNPushNotificationListenerService"
+            android:exported="false" >
+            <intent-filter>
+                <action android:name="com.google.android.c2dm.intent.RECEIVE" />
+            </intent-filter>
+        </service>
+        <meta-data
+            android:name="io.fabric.ApiKey"
+            android:value="${fabricKey}" />
```

Comment:
I don't have the permission to access the dashboard so added this value. To access, Just add the key and run the app. It will be directly added.